### PR TITLE
Add TinyPNG4Mac 1.0.3

### PR DESCRIPTION
**Pull request by L1cardo**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by reitermarkus** [comment]
The file is missing the `.rb` extension.

File: Casks/tinypng4mac [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'abstract' do
+  version '1.0.3'
+  sha256 '723a9e0b63b69a5b3c0d20cbe2199ff5dea7ac06ae511ec827b6bb82cf5af112'
+
+  url "https://github.com/kyleduo/TinyPNG4Mac/releases/download/v#{version}/TinyPNG4Mac_1_0_3.zip"
```
Comment (reitermarkus) [inline Casks/tinypng4mac]: Use `version.dots_to_underscores`.

**Comment by reitermarkus** [comment]
> - [x] `brew cask style --fix {{cask_file}}` reports no offenses.

I doubt that.

File: Casks/tinypng4mac [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cask 'abstract' do
```
Comment (reitermarkus) [inline Casks/tinypng4mac]: Wrong name.

**Comment by L1cardo** [comment]
image
image

**Comment by reitermarkus** [comment]
Seems like you don't have the same version locally as the one in this PR.

**Comment by L1cardo** [comment]
Can you rerun the `CI / ci` and the `Travis CI` ?

**Comment by reitermarkus** [comment]
It already did.

File: Casks/tinypng4mac.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  version '1.0.3'
+  sha256 '723a9e0b63b69a5b3c0d20cbe2199ff5dea7ac06ae511ec827b6bb82cf5af112'
+
+  url "https://github.com/kyleduo/TinyPNG4Mac/releases/download/v#{version}/TinyPNG4Mac_1_0_3.zip"
+  appcast 'https://github.com/kyleduo/TinyPNG4Mac/releases.atom'
+  name 'TinyPNG4Mac'
+  homepage 'https://github.com/kyleduo/TinyPNG4Mac'
+
+  app 'TinyPNG4Mac.app'
+
```
Comment (reitermarkus) [inline Casks/tinypng4mac.rb]: Remove the empty line.

**Comment by L1cardo** [comment]
Sorry to bothor you :(

**Comment by L1cardo** [comment]
I don't understand
image

**Comment by reitermarkus** [comment]
Please post the output of `cat ~/Desktop/tinypng4mac.rb`.

**Comment by L1cardo** [comment]
image

**Comment by L1cardo** [comment]
Oh my god! Finally!
Thank you very much! @reitermarkus

**Pull request merged**